source/server/http/admin.h

```diff
@@ -115,6 +115,7 @@ class AdminImpl : public Admin,
   void addOutlierInfo(const std::string& cluster_name,
                       const Upstream::Outlier::Detector* outlier_detector,
                       Buffer::Instance& response);
+  std::string statsAsJson(std::map<std::string, uint64_t> all_stats);
```

Comment:
oops sorry one more thing, can you make this `const std::map<std::string, uint64_t>&`. Sorry! :)

Reply:
No issues. committed..please check